pkg/kubelet/apis/config/validation/validation_linux.go

```diff
@@ -38,5 +40,13 @@ func validateKubeletOSConfiguration(kc *kubeletconfig.KubeletConfiguration) erro
 		return fmt.Errorf("invalid configuration: singleProcessOOMKill must not be explicitly set to false when using cgroup v1")
 	}
 
+	if userNs := kc.UserNamespaces; userNs != nil {
+		if idsPerPod := userNs.IDsPerPod; idsPerPod != nil {
+			if *idsPerPod%userNsUnitLength != 0 {
```

Comment:
also verify the field is >= userNsUnitLength

Reply:
done